XamarinCommunityToolkit/Behaviors/AttachedNumericValidationBehavior.shared.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Behaviors
+{
+    public static class AttachedNumericValidationBehavior
+    {
+        public static readonly BindableProperty AttachBehaviorProperty =
+            BindableProperty.CreateAttached("AttachBehavior", typeof(bool), typeof(AttachedNumericValidationBehavior), false, propertyChanged: OnAttachBehaviorChanged);
+
+        public static bool GetAttachBehavior(BindableObject view) 
+            => (bool)view.GetValue(AttachBehaviorProperty);
+
+        public static void SetAttachBehavior(BindableObject view, bool value) 
+            => view.SetValue(AttachBehaviorProperty, value);
+        
+        static void OnAttachBehaviorChanged(BindableObject view, object oldValue, object newValue)
+        {
+            var entry = view as Entry;
+
+            if (entry == null)
+                return;
+
+            var attachBehavior = (bool)newValue;
+
+            if (attachBehavior)
+            {
+                entry.TextChanged += OnEntryTextChanged;
+            }
+            else
+            {
+                entry.TextChanged -= OnEntryTextChanged;
+            }
+        }
+
+        static void OnEntryTextChanged(object sender, TextChangedEventArgs args)
+        {
+            double result;
+            var isValid = double.TryParse(args.NewTextValue, out result);
+
+            if (sender is Entry entry)
+            {
+                Color currentTextColor = entry.TextColor;
+                entry.TextColor = isValid ? currentTextColor : Color.Red;
```

Comment:
Can we make the invalid color configurable?

Reply:
Agree with this sentiment 👍

Reply:
I added the fixes you requested and pushed to my fork.  But it is not pushing to this PR.  I tried rebasing and everything is current.  Any reason why I cannot push to the PR?